tests/core/classic-snap16/task.yaml

```diff
@@ -4,6 +4,8 @@ summary: Ensure classic dimension works correctly
 # recommended instead of the classic snap
 systems: [ubuntu-core-16-*]
 
+tags: [ basic ]
```

Comment:
this one seems also fine to just run with PRs

Reply:
removed